src/aihwkit/nn/modules/conv.py

```diff
@@ -180,12 +180,14 @@ def forward(self, x_input: Tensor) -> Tensor:
             dilation=self.dilation,
             padding=self.padding,
             stride=self.stride,
-        ).transpose(1, 2)
+        ).transpose(-1, -2)
 
-        out = self.analog_module(x_input_).transpose(1, 2)
+        out = self.analog_module(x_input_).transpose(-1, -2)
```

Comment:
I hope this was tested in all possible input cases (in_trans, out_trans) and all tiles + indexed and non-indexed convolution? It does look rather wrong to me to use -1 -2 instead of 1, 2

Reply:
I run the tests with the change and everything seems to pass and look fine. Also, @jubueche runs the tests, and they work well for him.